rust/src/storage/file/mod.rs

```diff
@@ -121,7 +121,7 @@ impl StorageBackend for FileStorageBackend {
             Ok(_) => Ok(()),
             Err(e) => {
                 // If rename failed, clean up the temp file.
-                //self.delete_obj(tmp_path).await?;
+                // self.delete_obj(tmp_path).await?;
```

Comment:
Ugh, that's a leftover from https://github.com/delta-io/delta-rs/pull/376/files#diff-ce6ba0c9d799f33c7c8ec4b5d7f36e1c03c8451390190a348b530caf838c89dfR124, I commented it out in a test and forgot to clear

Reply:
@zijie0 Could you please revert that back?

Reply:
Done.